### PR TITLE
Qt: FMV tooltip expand + frame-time typo

### PR DESCRIPTION
**Pull request by RedDevilus**
### Description of Changes

~~This will add the Virtual in front of the controller name, in future we should change the struct so if you change to other type like Guitar, Pop n Music, or in future new pad devices to not say controllers.~~

Also added that FMV Aspect Ratio will inherent the value of the regular Aspect Ratio if you put the same value for Aspect Ratio and FMV Aspect ratio it would have the same effect if you left FMV Aspect Ratio on the 'Off (Default)'. Extra word with Override along side FMV Aspect Ratio as it will copy/match the general value and less rely on tooltip

Examples:

### Rationale behind Changes

~~Less confusing for players that the controller you see on PCSX2 controller port section is the one you emulate as, aka tricking the game you are using a DualShock 2 controller where it can actually be Keyboard to 8bitdo, Sony (DS3, DS4, DS5) , Xbox (360, One) and more ...~~

More correct and easier for people to know what FMV Aspect ratio does, also fix the typo for frametimes as it looks better.
### Suggested Testing Steps

Test tooltip

### CODE REVIEW

**Comment by Mrlinkwii** [comment]
This adds more confusion than it helps

**Comment by TheTechnician27** [comment]
Looks kind of scrunched in there, and I entirely agree with link that this only adds confusion where there wasn't any before or where it was extremely minimal.

**Comment by RedDevilus** [comment]
image

Changed it back that left stays same, just the title will stay say virtual.

**Comment by TheTechnician27** [comment]
I think this new change to the PR makes it somehow even more confusing to nearly every user, Red.

**Comment by Mrlinkwii** [comment]
Yeah what you changed managed to make it more confusing

File: pcsx2-qt/Settings/GraphicsSettingsWidget.cpp [hunk 1]
```diff
@@ -438,8 +438,9 @@ GraphicsSettingsWidget::GraphicsSettingsWidget(SettingsWindow* dialog, QWidget*
 		dialog->registerWidgetHelp(m_ui.PCRTCOverscan, tr("Show Overscan"), tr("Unchecked"),
 			tr("Enables the option to show the overscan area on games which draw more than the safe area of the screen."));
 
-		dialog->registerWidgetHelp(
-			m_ui.fmvAspectRatio, tr("FMV Aspect Ratio"), tr("Off (Default)"), tr("Overrides the full-motion video (FMV) aspect ratio."));
+		dialog->registerWidgetHelp(m_ui.fmvAspectRatio, tr("FMV Aspect Ratio"), tr("Off (Default)"),
+			tr("Overrides the full-motion video (FMV) aspect ratio.<br>"
+			   "If disabled aka default, it will inherent same value from above Aspect Ratio value which if also default is 'Auto Standard (4:3/3:2 Progressive)' or other value if changed there as well."));
```
Comment (F0bes) [inline pcsx2-qt/Settings/GraphicsSettingsWidget.cpp:443]: Way too wordy; took me a couple reads to understand what you're trying to say.
Cut the fat. Something like "If disabled, the FMV Aspect Ratio will be the same as the general Aspect Ratio" gets the point across.
The user already knows the defaults; the UI tells them.

Maybe even changing the setting to "FMV Aspect Ratio Override" makes more sense.

**Review by F0bes — COMMENTED** [review]
Any update on this?
Like others have said, appending virtual to everything might just make things confusing.
In some cases it doesn't make sense like "//: (Virtual) Controller Port is an official term from Sony."

File: pcsx2-qt/Settings/GraphicsSettingsWidget.cpp (hunk 1 above)
Comment (RedDevilus) [reply pcsx2-qt/Settings/GraphicsSettingsWidget.cpp:443]: I'll change it to be better for users in terms of readability.

Comment (RedDevilus) [reply pcsx2-qt/Settings/GraphicsSettingsWidget.cpp]: Okay changed it.

**Review by F0bes — APPROVED** [review]
